functions/src/modules/SE.functions.ts

```diff
@@ -88,3 +88,58 @@ export const processNewAllotment = functions.database
     });
     return 1;
   });
+
+const serialRegex = "^[a-zA-Z]+-\\d+";
+
+const updateChunk = async (path: string, taskId: string) => {
```

Comment:
I don't feel that extracting this function helps understanding the code where it is called. I suggest doing the update inline below.